src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -36,25 +36,20 @@ With [our Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricin
   src={accountsEditRetentionSettings}
 />
 <figcaption>
-  The <DoNotTranslate>**Data retention**</DoNotTranslate> UI is where you can see your data retention periods, and edit data retention. To view explanations of the table column headers, hover over a header. 
+  The <DoNotTranslate>**Data retention**</DoNotTranslate> UI is where you can view and edit your effective retention values, as well as upgrade your retention plan. To view explanations of the table column headers, hover over each header in the UI, or see our explanations in the **Requirements** section below.
 </figcaption>
 
 ## Requirements [#requirements]
 
-Being able to configure data retention requires Pro or Enterprise edition with billing administrator rights. Account characteristics that do not meet this criteria will block some New Relic organizations from adjusting retention on their own. If you're a Pro or Enterprise organization on a billing administrator account, and you can't edit retention, talk to your New Relic account representative.
+To upgrade your retention plan, or edit your effective retention values when you have more than one account in your organization, you need to be on the Pro or Enterprise edition. With these editions, you can designate users who are responsible for managing the retention plan or the effective amount of time data will be retained. Whether you have one user handle both tasks or assign separate users for each, you enable access differently:
```

Comment:
@rhetoric101 Can we clean up some of the more complex wording to simplify it? For example, "designate" --> "choose"